### PR TITLE
chore: update golang docker image to 1.6-alpine

### PR DESCRIPTION
**Pull request by nadade**

### CODE REVIEW

**Comment by abtris** [comment]
I this ready?

File: Dockerfile [hunk 1]
```diff
@@ -1,8 +1,10 @@
...
+FROM golang:1.6-alpine
 
 ENV GO15VENDOREXPERIMENT 1
 ENV VERSION 1.13.1
 
+RUN apk add --update git
+
 ADD . /go/src/github.com/apiaryio/dnsdock
 
 ADD https://github.com/Masterminds/glide/releases/download/0.9.1/glide-0.9.1-linux-amd64.tar.gz /tmp/glide-0.9.1-linux-amd64.tar.gz
```
Comment (abtris) [inline Dockerfile]: Maybe good to update glide, latest version is  0.12.3

**Comment by nadade** [comment]
shrank the image even more and bumped glide version to 0.12.3

**Comment by abtris** [comment]
Tests failed

**Comment by nadade** [comment]
@abtris resolved

**Pull request merged**